src/main/resources/api.yml

```diff
@@ -0,0 +1,503 @@
+      The response includes the name and other details about the business.
+      parameters:
+      - name: id
+        in: url
+        description: Profile id
+        required: true
+        type: string
+      security:
+      - token: []
+      tags:
+      - business profile
+      responses:
+        "200":
+          description:  Profile fields
+          schema:
+            type: json
+            profile:
+              $ref: '#/definitions/Business'
+        "403":
+          description: Unauthorized for the current operation
+          schema:
+            $ref: '#/definitions/Error'
+        "404":
+          description: Profile not found
+          schema:
+            $ref: '#/definitions/Error'
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+    /orders:
+      get:
+        summary: Existing orders
+        description: The endpoint returns information about orders assigned to this business.
+        The response includes the order details, and lists the orders according to their execution time.
+        parameters:
+        - name: business id
+          in: query
+          description: id of the current business
+          required: true
+          type: string
+        security:
+        - token: []
+        tags:
+        - orders
+        responses:
+          "200":
+            description: An array of orders
+            schema:
+              type: array
+              orders:
+                $ref: '#/definitions/Order'
+          default:
+            description: Unexpected error
+            schema:
+              $ref: '#/definitions/Error'
+
+      post:
+        summary: Adds a new order
+        description: The endpoint sends details of an order provided by the client, containing content of the following fields: name, phone number, e-mail, date, time, number of participants and additional order information.
+                       The response includes the result of the operation.
+        requestBody:
+          content:
+            application/json:     # Media type
+              schema:             # Request body contents
+                $ref: '#/components/schemas/Order'   # Reference to an object
+              examples:    # Child of media type
+                Lena:   # Example 1
+                  value:
+                    name: Lena
+                    phone: +49 176 68558497
+                    eMail: aoneko@gmx.de
+                    date: 27.09.2018
+                    time: 15:00-20:00
+                    numberParticipants: 1
+                    description: Hi, I want a dog sled tour in Berlin.
+        responses:
+          "200":
+            description: Result of the operation
+            schema:
+              type: long
+
+          default:
+            description: Unexpected error
+            schema:
+              $ref: '#/definitions/Error'
+
+      /order/{id}:
+        get:
```

Comment:
Then this one will look like:
`/business/{business_id}/orders/{order_id}`

Reply:
same here